Package.swift

```diff
@@ -3,183 +3,188 @@
 import Foundation
 import PackageDescription
 
-enum CombineImplementation: CaseIterable {
+// MARK: Helpers
+let env = ProcessInfo.processInfo.environment
+
+extension Optional where Wrapped: RangeReplaceableCollection {
+    
+    mutating func append(contentsOf newElements: [Wrapped.Element]) {
+        if newElements.isEmpty { return }
+        
+        if let wrapped = self {
+            self = wrapped + newElements
+        } else {
+            self = .init(newElements)
+        }
+    }
+}
+
+// MARK: Package
+let package = Package(
+    name: "CombineX",
+    platforms: [
+        .macOS(.v10_10), .iOS(.v8), .tvOS(.v9), .watchOS(.v2),
+    ],
+    products: [
+        .library(name: "CombineX", targets: ["CombineX", "CXFoundation"]),
+        .library(name: "CXShim", targets: ["CXShim"]),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/Quick/Quick.git", from: "2.0.0"),
+        // TODO: Use "8.0.2" until https://github.com/Quick/Nimble/issues/705 is fixed.
+        .package(url: "https://github.com/Quick/Nimble.git", .exact("8.0.2")),
+    ],
+    targets: [
+        .target(name: "CXLibc"),
+        .target(name: "CXUtility"),
+        .target(name: "CXNamespace"),
+        .target(name: "CombineX", dependencies: ["CXLibc", "CXUtility", "CXNamespace"]),
+        .target(name: "CXFoundation", dependencies: ["CXUtility", "CXNamespace", "CombineX"]),
+        .target(name: "CXCompatible", dependencies: ["CXNamespace"]),
+        .target(name: "CXShim", dependencies: [/* depends on combine implementation */]),
+        .testTarget(name: "CombineXTests", dependencies: ["CXUtility", "CXShim", "Quick", "Nimble"]),
+        .testTarget(name: "CXFoundationTests", dependencies: ["CXShim", "Quick", "Nimble"]),
+        .testTarget(name: "CombineXFailingTests", dependencies: ["CXShim", "Quick", "Nimble"]),
```

Comment:
`CombineXFailingTests` or `CXFailingTests`?

This target should not be tested by CI.